src/widgets/Header/lib/account-dropdown-options.constant.ts

```diff
@@ -1,19 +1,28 @@
 import { useNavigate } from "react-router-dom"
 
+import { activityModel } from "~/entities/activity"
 import { useLogoutMutation, userModel } from "~/entities/user"
 import { ROUTES, secureTokensStorage } from "~/shared/utils"
 
 export const useAccountDropdown = () => {
   const navigate = useNavigate()
   const { clearUser } = userModel.hooks.useUserState()
+  const { clearActivityInProgressState } = activityModel.hooks.useActivityClearState()
 
-  const { mutate: logout, isLoading } = useLogoutMutation({
-    onSuccess() {
-      clearUser()
-      secureTokensStorage.clearTokens()
-      navigate(ROUTES.login.path)
-    },
-  })
+  const { mutate: logoutMutation, isLoading } = useLogoutMutation()
+
+  const logout = () => {
+    const tokens = secureTokensStorage.getTokens()
+
+    if (tokens?.accessToken) {
+      logoutMutation({ accessToken: tokens.accessToken })
+    }
+
+    clearUser()
+    clearActivityInProgressState()
+    secureTokensStorage.clearTokens()
+    navigate(ROUTES.login.path)
+  }
```

Comment:
This should be in a scope of the logout feature

Reply:
Done